packages/app-layout/src/vaadin-app-layout.js

```diff
@@ -121,6 +121,7 @@ class AppLayout extends ElementMixin(ThemableMixin(ControllerMixin(PolymerElemen
           height: 100%;
           --vaadin-app-layout-transition: 200ms;
           transition: padding var(--vaadin-app-layout-transition);
+          --vaadin-app-layout-drawer-width: 16em;
```

Comment:
There shouldn't be a default value for the `--vaadin-app-layout-drawer-width` property on the host because it would override any value applied to the property somewhere higher in the hierarchy.

For example, the following wouldn't work:

```css
body {
  --vaadin-app-layout-drawer-width: 150px;
}
```

Reply:
Changed to define a private CSS prop that takes the `--vaadin-app-layout-drawer-width` and has a fallback value.